lib/optimizely/decision_service.rb

```diff
@@ -113,7 +112,7 @@ def get_variation_for_feature(feature_flag, user_id, attributes = nil)
       # user_id - String ID for the user
       # attributes - Hash representing user attributes
       #
-      # Returns hash with the experiment and variation where visitor will be bucketed (nil if the user is not bucketed into any of the experiments on the feature)
+      # Returns decision with the experiment and variation where visitor will be bucketed (nil if the user is not bucketed into any of the experiments on the feature)
```

Comment:
Please replace this message with below: 
Returns Decision struct  (nil if the user is not bucketed into any of the experiments on the feature)